openfisca_us/variables/fcc/lifeline/broadband_cost_after_lifeline.py

```diff
@@ -0,0 +1,16 @@
+from openfisca_us.model_api import *
+
+
+class broadband_cost_after_lifeline(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Broadband costs after Lifeline"
+    documentation = "Broadband costs after Lifeline benefits"
+    definition_period = YEAR
+    unit = USD
+    reference = "https://www.law.cornell.edu/cfr/text/47/54.403"
+
+    def formula(spm_unit, period, parameters):
+        broadband = spm_unit("broadband_cost", period)
+        lifeline = spm_unit("lifeline", period)
+        return max_(broadband - lifeline, 0)
```

Comment:
Should we add a `positive(x) -> max_(x, 0)` function in openfisca-tools?

Reply:
I'd prefer not, for the same reasons removing `amount_over` (https://github.com/PolicyEngine/openfisca-tools/issues/13). But feel free to add more rationale in an issue if you'd like.